### PR TITLE
[WIP] Help for candidates and committees landing and getting started homepage section

### PR DESCRIPTION
**Pull request by patphongs**
This is the new PR after closing #1290 in order to clean things up a little. 

This is the new getting started section on the homepage.

screen shot 2017-11-02 at 4 47 55 pm

screen shot 2017-11-03 at 10 41 51 am

### CODE REVIEW

**Comment by johnnyporkchops** [comment]
I am requesting a hi-res render of the contributions image from @jenniferthibault for the contributions section.

**Comment by jenniferthibault** [comment]
@johnnyporkchops here's a web-ready version of the contributions image in narrow format. (Not high-resolution, but exported for web from the original....which I think is what you need?) If you do need high resolution, glad to get it.

I _think_ I matched the naming convention that I found in this PR, but feel free to fix it if not.
[feature--contributions-narrow.jpg.zip](https://github.com/18F/fec-cms/files/1442407/feature--contributions-narrow.jpg.zip)

File: fec/fec/static/scss/_variables.scss [hunk 1]
```diff
@@ -76,3 +76,13 @@ $z-header: $z6;
 $z-sticky: $z7;
 $z-glossary: $z8;
 $z-feedback: $z-max;
+
+// Neat Breakpoints
```
Comment (xtine) [inline fec/fec/static/scss/_variables.scss]: Are these still defined in `scss/layout/_grid.scss`? If so we need to remove those if we are going to define these variables here instead.

**Comment by xtine** [comment]
Can we remove `statement_of_organization_form.png`?

File: fec/fec/static/scss/layout/_grid.scss [hunk 2]
```diff
@@ -72,7 +72,34 @@
...
+.grid__icon--deep-red {
+  img {
+    background-color:$deep-red;
+ }
+}
+
+.grid__icon--navy {
+  img {
+    background-color:$navy;
+ }
```
Comment (xtine) [inline fec/fec/static/scss/layout/_grid.scss]: check indent

File: fec/fec/static/scss/layout/_grid.scss [hunk 3]
```diff
@@ -72,7 +72,34 @@
...
+    height: u(15rem);
+    object-fit: contain;
+    padding: u(2rem);
+    width: 100%;
+    }
+  }
+
+.grid__icon--deep-red {
+  img {
+    background-color:$deep-red;
```
Comment (xtine) [inline fec/fec/static/scss/layout/_grid.scss]: Space after `:`

File: fec/fec/static/scss/layout/_grid.scss [hunk 4]
```diff
@@ -72,7 +72,34 @@
...
+ }
+}
+
+.grid__icon--navy {
+  img {
+    background-color:$navy;
```
Comment (xtine) [inline fec/fec/static/scss/layout/_grid.scss]: Space after `:`

File: fec/home/templates/home/candidate-and-committee-services/services_landing_page.html [hunk 5]
```diff
@@ -1,110 +1,152 @@
...
-        </div>
+<div class="container main">
+  <div class="container">
+    <h2>Guides</h2>
+    <div class="grid grid--4-wide">
+      <div class="grid__item">
+        <a href="/help-candidates-and-committees/guides/?tab=candidates-and-their-authorized-committees">
+          <aside class="card card--horizontal card--secondary">
+            <div class="card__image__container">
+                <span class="card__icon i-notebook"><span class="u-visually-hidden">Icon representing a notebook</span></span>
```
Comment (xtine) [inline fec/home/templates/home/candidate-and-committee-services/services_landing_page.html]: Check this indent

File: fec/home/templates/home/candidate-and-committee-services/services_landing_page.html [hunk 6]
```diff
@@ -1,110 +1,152 @@
...
-          <span>All FEC registration and reporting forms</span>
-        </div>
+      <div class="grid__item">
+        <a href="/help-candidates-and-committees/guides/?tab=political-party-committees">
+          <aside class="card card--horizontal card--secondary">
+            <div class="card__image__container">
+              <span class="card__icon i-notebook"><span class="u-visually-hidden">Icon of a training screen</span></span>
+            </div>
+            <div class="card__content">
+                Political party committees
```
Comment (xtine) [inline fec/home/templates/home/candidate-and-committee-services/services_landing_page.html]: Check this indent

File: fec/home/templates/home/candidate-and-committee-services/services_landing_page.html [hunk 7]
```diff
@@ -1,110 +1,152 @@
...
-      <div class="message__content">
-        <p class="u-no-margin">We're actively reorganizing our content about candidates and committees. If you can't find what you're looking for, you can try browsing this content in <a href="{{ settings.FEC_TRANSITION_URL }}/info/compliance.shtml">its previous form</a>.</p>
+      <div class="grid__item">
+        <a href="/help-candidates-and-committees/guides/?tab=corporations-and-labor-organizations">
+          <aside class="card card--horizontal card--secondary">
+            <div class="card__image__container">
+              <span class="card__icon i-notebook"><span class="u-visually-hidden">Icon of a candidate</span></span>
+            </div>
+            <div class="card__content">
+                Corporations and labor organizations
```
Comment (xtine) [inline fec/home/templates/home/candidate-and-committee-services/services_landing_page.html]: Check this indent

File: fec/home/templates/home/candidate-and-committee-services/services_landing_page.html [hunk 8]
```diff
@@ -1,110 +1,152 @@
...
+        </a>
+      </div>
+      <div class="grid__item">
+        <a href="/help-candidates-and-committees/guides/?tab=political-action-committees">
+          <aside class="card card--horizontal card--secondary">
+            <div class="card__image__container">
+              <span class="card__icon i-notebook"><span class="u-visually-hidden">Icon of a candidate</span></span>
+            </div>
+            <div class="card__content">
+                Political action committees (PACs)
```
Comment (xtine) [inline fec/home/templates/home/candidate-and-committee-services/services_landing_page.html]: Check this indent

File: fec/home/templates/home/candidate-and-committee-services/services_landing_page.html [hunk 9]
```diff
@@ -1,110 +1,152 @@
...
+      </li>
+      <li class="grid__item grid__item--with-img">
+        <img src="{% static 'img/feature--training.jpg' %}" alt="">
+        <h3 class="u-margin--top"><a href="https://transition.fec.gov/info/outreach.shtml">Education and learning</a></h3>
+        <p>Videos, trainings, webinars and conferences</p>
+      </li>
+    </ul>
+  </div>
+</div>
+ <section class="slab slab--neutral">
```
Comment (xtine) [inline fec/home/templates/home/candidate-and-committee-services/services_landing_page.html]: Check this indent spacing

File: fec/home/templatetags/tips_for_treasurers.py [hunk 10]
```diff
@@ -0,0 +1,17 @@
+import re
+
+from django import template
+from django.conf import settings
+from home.models import TipsForTreasurersPage
+
+register = template.Library()
+
```
Comment (xtine) [inline fec/home/templatetags/tips_for_treasurers.py]: Need an extra line for spacing.

File: fec/home/templatetags/tips_for_treasurers.py [hunk 11]
```diff
@@ -0,0 +1,17 @@
...
+@register.inclusion_tag('partials/tips-for-treasurers.html')
+def tips_for_treasurers_feed():
+    # get one of each update type (not featured)
+
+    tips_for_treasurers=TipsForTreasurersPage.objects.live().order_by('-date')[:4]
```
Comment (xtine) [inline fec/home/templatetags/tips_for_treasurers.py]: Space between `=`

File: fec/home/templatetags/tips_for_treasurers.py [hunk 12]
```diff
@@ -0,0 +1,17 @@
...
+
+    return {
+        'tips_for_treasurers': tips_for_treasurers
+    }
```
Comment (xtine) [inline fec/home/templatetags/tips_for_treasurers.py]: Need a newline at end of file. Please run this file through a PEP8 checker.

File: fec/home/templatetags/home_page.py [hunk 13]
```diff
@@ -50,4 +50,4 @@ def home_page_news():
 
     return {
         'updates': updates
-    }
+    }
```
Comment (xtine) [inline fec/home/templatetags/home_page.py]: This needs a newline at end of file.

File: fec/home/templates/partials/tips-for-treasurers.html [hunk 14]
```diff
@@ -0,0 +1,9 @@
+{% load filters %}
+<ul class="list--spacious">
+  {% for tips_for_treasurer in tips_for_treasurers %}
```
Comment (xtine) [inline fec/home/templates/partials/tips-for-treasurers.html]: I might suggest a variable like `tip` to make the variables more concise.

File: fec/fec/static/scss/_variables.scss (hunk 1 above)
Comment (patphongs) [reply fec/fec/static/scss/_variables.scss]: The seem to be in `scss/_grid.scss`. So we removed the reference here.

**Comment by patphongs** [comment]
We made all the necessary changes for this PR as has been noticed. Merging this in now so that we can get it in time for testing at the conference. If there are any additional fixes needed, we can do it in dev before the next release. Thanks!

**Comment by johnnyporkchops** [comment]
Flagging this to go along with any additional changes to this after testing and before pushing to production: The "All tips" button needs styling adjustment: Add "--alt" to first class listed, so it reads:
`<a class="button--alt button--updates heading__right" href="/updates/?update_type=tips-for-treasurers">All tips</a> `

**Pull request merged**